terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -764,6 +767,15 @@ def qq(*words):
             }
             for zone in range(num_zones)
         },
+        'aws_default_security_group': {
+            'default': {
+                # CIS 4.3 Ensure the default security group of every VPC restricts all traffic
+                # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html#securityhub-cis-controls-4.3
+                'vpc_id': '${aws_default_vpc.default.id}',
```

Comment:
This is the default SG for the default VPC. What about the default SG of the `gitlab` VPC?